src/sql/parts/notebook/outputs/renderers.ts

```diff
@@ -0,0 +1,630 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+import { default as AnsiUp } from 'ansi_up';
+//import AnsiUp from 'typings/modules/ansi_up/ansi_up';
```

Comment:
Please remove this line, looks like it was missed when you fixed the imports issue

Reply:
Done